FHEM/00_SIGNALduino.pm

```diff
@@ -2921,7 +2923,26 @@ sub SIGNALduino_Attr(@) {
 		return "Note, please use other userReadings names.\nReserved names from $name are: cc1101_config, cc1101_config_ext, cc1101_patable"
 			if ($aVal =~ /cc1101_(?:config(?:_ext)?|patable)(?:\s|{)/);
 	}
- 	return ;
+	elsif ($aName eq "rfmode")	# change receive mode
+	{
+		my $oldAttrib = AttrVal($name, 'rfmode', 'SlowRF');
+
+		if ($aVal ne $oldAttrib) {
+			$hash->{logMethod}->($name, 3, "$name: Attr, $aName set to $aVal (please check protocollist)");
+			if ($aVal ne 'SlowRF') {
+				my ($manu, $adr) = cc1101_manspecific::Register($aVal);         # return array with cc1101 manufacturer specific´s register settings
+				for my $i (0...scalar(@{$manu})-1) {
+					$hash->{logMethod}->($name, 5, "$name: Attr, $aName - write adress " . @{$adr}[$i] . " -> " . @{$manu}[$i]);
+					my $adrOFFset = sprintf("%02X", hex(@{$adr}[$i]) + 2 );
```

Comment:
vermutlich  könnte wir die Adresse vom Register gleich dezimal und mit offset speichern und uns das umwandeln in dezimal ersparen oder?

Reply:
Das könnte man lösen ABER dann nehme ich oben drüber ein kommentarfeld mit den Adressen ohne offset. Es soll ja auf den Ersten Blick immer wiederkehrend nachvollziehbar sein.

Der markierte Punkt von dir, war der Grund wieso der Radino oder Nano "verstummen". Ich überdachte die Schreibbefehle in einem Rutsch zu schreiben und nicht nacheinader ;-)